codechain/config/mod.rs

```diff
@@ -325,6 +345,23 @@ impl Stratum {
     }
 }
 
+impl ShardValidator {
+    pub fn overwrite_with(&mut self, matches: &clap::ArgMatches) -> Result<(), String> {
+        if matches.is_present("no-shard-validator") {
+            self.disable = true;
+        }
+
+        if let Some(account) = matches.value_of("shard-validator") {
+            self.account = Some(Address::from_str(account).map_err(|_| "Invalid address")?);
```

Comment:
Why don't you use `parse_address()` here? It supports 0x-prefixed addresses.

Reply:
I'll update it.